test/parallel/test-fs-opendir.js

```diff
@@ -223,12 +223,10 @@ async function doAsyncIterInvalidCallbackTest() {
 }
 doAsyncIterInvalidCallbackTest().then(common.mustCall());
 
-// Check if directory already closed - throw an exception
+// Check first call to close() - should not report an error.
 async function doAsyncIterDirClosedTest() {
   const dir = await fs.promises.opendir(testDir);
   await dir.close();
```

Comment:
I think we should also add back the test that a second call raises an error:
```suggestion
  await dir.close();
  
  await assert.rejects(() => dir.close(), dirclosedError);
```